seed/challenges/01-front-end-development-certification/basic-javascript.json

```diff
@@ -1159,7 +1159,8 @@
       "id": "56533eb9ac21ba0edf2244b6",
       "title": "Escape Sequences in Strings",
       "description": [
-        "Quotes are not the only characters that can be <dfn>escaped</dfn> inside a string. Here is a table of common escape sequences:",
+        "Quotes are not the only characters that can be <dfn>escaped</dfn> inside a string. There are two reasons to escaping characters. First is to allow you to use characters you might not otherwise be able to type out, such as a backspace. Second is to allow you to represent multiple quotes in a string without JavaScript misinterpreting what you mean. We learned this in the previous challenge.",
```

Comment:
~~There are two reasons to escaping characters.~~ There are two reasons to use escaping characters.